resources/assets/js/features/public-profiles/reducer.js

```diff
@@ -3,22 +3,25 @@ import * as actions from './actionTypes';
 const initialState = {
     current_driver_profile: {},
     current_passenger_profile: {},
-    preloader: true
+    isFetched: null
```

Comment:
use same naming style if you started with underscore state property naming style use underscore